dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/utils/Assertions.java

```diff
@@ -117,6 +118,18 @@ public static <E> void assertIsEmpty( Collection<E> actual )
         assertTrue( actual.isEmpty(), actual.toString() );
     }
 
+    /**
+     * Asserts that the given collection is not null and not empty.
+     *
+     * @param <E>
+     * @param actual the collection.
+     */
+    public static <E> void assertNotEmpty( Collection<E> actual )
```

Comment:
If a type parameter is not needed for consistent type checking it is better to use `Collection<?>` instead.

Reply:
Agree, fixed.